sql/2021/mobile-web/client_hints.sql

```diff
@@ -0,0 +1,62 @@
+# standardSQL
+# Usage of client hint directives
+CREATE TEMPORARY FUNCTION getClientHints(payload STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS """
+try {
+  const parsed_headers = JSON.parse(headers);
+  const matching_headers = parsed_headers.filter(h => h.name.toLowerCase() == headername.toLowerCase());
```

Comment:
@obto don't think this function is defined correctly as you pass the `payload` but then use `headers` and `headername`.

```suggestion
CREATE TEMPORARY FUNCTION getClientHints(headers STRING, headername STRING)
RETURNS ARRAY<STRING> LANGUAGE js AS """
try {
  const parsed_headers = JSON.parse(headers);
  const matching_headers = parsed_headers.filter(h => h.name.toLowerCase() == headername.toLowerCase());
```

Reply:
Yup, thats gotta be it